controller/DeliveryServer.php

```diff
@@ -352,6 +331,38 @@ public function finishDeliveryExecution()
         $this->redirect($this->getReturnUrl());
     }
 
+    protected function setTemplate(string $template, string $title)
```

Comment:
As per Least privilege principle, better to keep it private unless there is a good reason to make it protected.

```suggestion
    private function setTemplate(string $template, string $title)
```